File: Sample Applications/CarouselImageGallery/CarouselImageGalleryCS/GalleryLauncher.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using Telerik.WinControls.UI;

namespace CarouselImageGallery
{
    public partial class GalleryLauncher : Form
    {
        public GalleryLauncher()
        {
            InitializeComponent();
        }

        private void radButton2_Click(object sender, EventArgs e)
        {
            FolderBrowserDialog fbd = new FolderBrowserDialog();
            if (!string.IsNullOrEmpty(this.radDropDownList1.Text))
            {
                fbd.SelectedPath = this.radDropDownList1.Text;
            }

            if (fbd.ShowDialog() == DialogResult.OK)
            {
                this.radDropDownList1.Text = fbd.SelectedPath;
            }
        }

        private void btnBrowse_Click(object sender, EventArgs e)
        {
            string path = this.radDropDownList1.Text.Trim();
            if (this.radDropDownList1.FindItemExact(path, false) == null)
            {
                this.radDropDownList1.Items.Add(new RadListDataItem(path));
            }

            GalleryView galleryView = new GalleryView();

            galleryView.ShowGallery(path);
        }

        private void GalleryLauncher_Load(object sender, EventArgs e)
        {
            this.radDropDownList1.Text = System.Environment.GetFolderPath(Environment.SpecialFolder.MyPictures);
        }
    }
}

File: Sample Applications/CarouselImageGallery/CarouselImageGalleryVB/GalleryLauncher.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports Telerik.WinControls.UI

Namespace CarouselImageGallery
	Public Partial Class GalleryLauncher
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub radButton2_Click(ByVal sender As Object, ByVal e As EventArgs) Handles radButton2.Click
			Dim fbd As FolderBrowserDialog = New FolderBrowserDialog()
			If (Not String.IsNullOrEmpty(Me.radDropDownList1.Text)) Then
				fbd.SelectedPath = Me.radDropDownList1.Text
			End If

			If fbd.ShowDialog() = System.Windows.Forms.DialogResult.OK Then
				Me.radDropDownList1.Text = fbd.SelectedPath
			End If
		End Sub

		Private Sub btnBrowse_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnBrowse.Click
			Dim path As String = Me.radDropDownList1.Text.Trim()
			If Me.radDropDownList1.FindItemExact(path, False) Is Nothing Then
				Me.radDropDownList1.Items.Add(New RadListDataItem(path))
			End If

			Dim galleryView As GalleryView = New GalleryView()

			galleryView.ShowGallery(path)
		End Sub

		Private Sub GalleryLauncher_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
			Me.radDropDownList1.Text = System.Environment.GetFolderPath(Environment.SpecialFolder.MyPictures)
		End Sub
	End Class
End Namespace